data/phones/list_phones.py

```diff
@@ -62,8 +65,25 @@ def main(args: argparse.Namespace):
             f"{len(examples):<10}"
             f"{', '.join(_pick_examples_for_display(examples))}"
         )
+        if not ipapy.is_valid_ipa(phone):
+            invalid_phones.add(phone)
     print(f"\n# unique phones: {len(phone_to_examples)}")
 
+    # Check the phoneme inventory for invalid IPA representations.
+    if len(invalid_phones) and args.filepath.endswith("phonemic.tsv"):
+        print(f"--- WARNING: {len(invalid_phones)} Invalid phonemes:")
```

Comment:
Seems like this ought to use logging.warning or something like that?

Reply:
I am wondering whether this logic should be dependent on the command-line flag, something along the lines of --check_phonemes. In which case you guys decide whether you want to use absl-py in this project.

Reply:
We use good ole `argparse` around here...but I don't think it needs to be configurable, so much as "future-configurable", hence the logging suggestion.

Reply:
Ok. I've implemented it the stone-age way. Depending on absl-py and producing warning.